l10n_es_account_invoice_sequence/models/account_journal.py

```diff
@@ -23,7 +23,7 @@
 from openerp import models, fields, api, exceptions, _
```

Comment:
Because I added as I need them, so I don't respect any preset order. It's so important? The list is very short